tests/queries/0_stateless/01017_mutations_with_nondeterministic_functions_zookeeper.sh

```diff
@@ -35,9 +51,21 @@ ${CLICKHOUSE_CLIENT} --query "ALTER TABLE $T1 DELETE WHERE rand() = 0" 2>&1 > /d
 ${CLICKHOUSE_CLIENT} --query "ALTER TABLE $T1 UPDATE y = y + rand() % 1 WHERE not ignore()" 2>&1 > /dev/null \
 && echo 'OK' || echo 'FAIL'
 
+# hm... it looks like joinGet condidered determenistic
+${CLICKHOUSE_CLIENT} --query "ALTER TABLE $R1 UPDATE y = joinGet('${CLICKHOUSE_DATABASE}.lookup_table', 'y_new', y) WHERE x=1" 2>&1 \
```

Comment:
Hm, but why, maybe it should be marked as non-deterministic instead? Objections?